server/controllers/notificationController.ts

```diff
@@ -35,24 +35,12 @@ export default class NotificationController {
     res.redirect('back')
   }
 
-  async getManage(req: Request, res: Response) {
-    const {
-      user: { employeeName, token },
-      authUrl,
-    } = req
-
-    const {
-      locals: { csrfToken, errors = null },
-    } = res
+  async getManage({ user: { token } }: Request, res: Response) {
     const { snoozeUntil, notificationsEnabled } = await this.getSnoozeAndNotificationSettings(token)
 
-    return res.render('pages/manage-your-notifications', {
-      errors,
```

Comment:
`errors` wasn't needed on this page anyway.
Rest are removed as they are in `res.locals` so don't need explicitly copying across